clarity/src/vm/docs/mod.rs

```diff
@@ -2001,7 +2001,7 @@ const DEFINE_TRAIT_API: DefineAPI = DefineAPI {
 can implement a given trait and then have their contract identifier being passed as a function argument in order to be called
 dynamically with `contract-call?`.
 
-Traits are defined with a name, and a list functions, defined with a name, a list of argument types, and return type.
+Traits are defined with a name, a list of functions, and a return type. The list of functions here is defined with a name and the list of argument types.
```

Comment:
Hi @tippenein. Thanks for trying to make this more clear! The return type should go with the functions. Would something like this be more clear?

```
Traits are defined with a name, and a list functions, where each function is defined with a name, a list of argument types, and a return type.
```

Reply:
Yeah that's better imo